test/testobjects/testhistories.ts

```diff
@@ -1352,7 +1352,8 @@ export class TestHistories {
             })
         );
 
-        timestamp = moment(timestamp).add(retryInterval, "ms").toDate();
+        let fireAt = moment(timestamp).add(retryInterval, "ms").toDate();
+        timestamp = moment(fireAt).add(10, "s").toDate();

```

Comment:
My only nitpick is to add a comment to these artificial delays explaining that you're adding these extra seconds because, in reality, timers don't fire exactly at the same time as their `fireAt` value.

Reply:
Done. 

Just FYI, there are other places where test histories are generated with timers, and those also have their `timestamp` the same as their `fireAt` value. It might be worthwhile to go and add artificial delays to those too, but I feel like that's outside the scope of this PR